spectractor/extractor/spectrum.py

```diff
@@ -553,14 +590,20 @@ def save_spectrogram(self, output_file_name, overwrite=False):
         hdu.writeto(output_file_name, overwrite=overwrite)
         self.my_logger.info('\n\tSpectrogram saved in %s' % output_file_name)
 
-    def load_spectrum(self, input_file_name):
+    def load_spectrum(self, input_file_name, spectrogram_file_name_override, psf_file_name_override):
```

Comment:
Are the new arguments mandatory?  They're optional in the `__init__` method.

Reply:
Nope, shouldn't be, that's a great catch, thanks!